### ISSUE TITLE
Author onboarding experience docs

### ISSUE THREAD

**Issue by jcklpe** [open-austin/practice-designers#2]
- [x] create github issue templates for roles
  - [Hack for LA example board](https://github.com/orgs/hackforla/projects/67)
- [x] create github issue template for experience profile
  - hackforla/UI-UX#577 
- [x] create github issue template for onboarding (??)
- [x] create project board for open roles across Open Austin and define process for keeping updated. 
- [Hack for LA UX Practice board](https://github.com/orgs/hackforla/projects/70)

**Comment by jcklpe** [comment]
create github issue templates for roles

create github issue template for experience profile

**Comment by jcklpe** [comment]
I created an issue template for people to create their own Contributor Profiles, similar to the "Experience Profile" that Hack for LA has.   
  
I separated out the onboarding materials into a second issue template for onboarding. Will need to chat with @lianilychee  in order to figure out what exactly that stuff needs to look like.   
  
Also created a template for open roles. Generally speaking the CoP won't have many open roles but this can serve as a guide for other teams to create their own open role templates. Some information will probably need to be adjusted based on feedback. It's just a quick first draft on my part.
